localization/ar_tag_based_localizer/launch/ar_tag_based_localizer.launch.xml

```diff
@@ -0,0 +1,21 @@
+<launch>
+  <arg name="param_file" default="$(find-pkg-share ar_tag_based_localizer)/config/ar_tag_based_localizer.param.yaml"/>
+  <arg name="node_name" default="ar_tag_based_localizer"/>
+
+  <!-- Topic names -->
+  <arg name="input_image" default="~/input/image"/>
+  <arg name="input_camera_info" default="~/input/camera_info"/>
+
+  <arg name="output_image" default="~/debug/result"/>
+  <arg name="output_pose_with_covariance" default="~/output/pose_with_covariance"/>
+
+  <node pkg="ar_tag_based_localizer" exec="ar_tag_based_localizer" name="$(var node_name)" output="log">
+    <remap from="image" to="$(var input_image)"/>
+    <remap from="camera_info" to="$(var input_camera_info)"/>
+
+    <remap from="result" to="$(var output_image)"/>
+    <remap from="pose_with_covariance" to="$(var output_pose_with_covariance)"/>
```

Comment:
Please fix here as well :+1: 
```suggestion
    <remap from="~/input/image" to="$(var input_image)"/>
    <remap from="~/input/camera_info" to="$(var input_camera_info)"/>

    <remap from="~/debug/result" to="$(var output_image)"/>
    <remap from="~/output/pose_with_covariance" to="$(var output_pose_with_covariance)"/>
```

Reply:
Fixed with the below commits.
https://github.com/autowarefoundation/autoware.universe/pull/4347/commits/408b12514fe249b844d97e6ab710ab679578c390
https://github.com/autowarefoundation/autoware.universe/pull/4347/commits/975fb1ca38944f6f8175227b91ceb2108f6e3a6b
I have confirmed that the standalone tool using this package works normally.